derp/derp_server.go

```diff
@@ -689,6 +690,36 @@ func (s *Server) unregisterClient(c *sclient) {
 	}
 }
 
+// addPeerGoneFromRegionWatcher adds a function to be called when peer
+// is gone from the region overall. It returns a handle that can be
+// used to remove the watcher later.
+func (s *Server) addPeerGoneFromRegionWatcher(peer key.NodePublic, f func(key.NodePublic)) set.Handle {
```

Comment:
it's nice to add `typically f is [sclient.onPeerGoneFromRegion]` or similar, as then there's a navigable link when someone is reading.

Reply:
done